src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj

```diff
@@ -64,7 +64,7 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Authorization.2.0.0\lib\net40\Microsoft.Azure.Management.Authorization.dll</HintPath>
```

Comment:
@hyonholee Can you please add a changelog entry in the compute changelog for these changes?  Otherwise, this looks good.

Reply:
Done.  Thanks.